src/opera/test/pge/disp_s1/test_disp_s1_pge.py

```diff
@@ -118,6 +150,104 @@ def test_disp_s1_pge_execution(self):
 
         self.assertIn(f"DISP-S1 invoked with RunConfig {expected_sas_config_file}", log_contents)
 
+    def test_disp_s1_pge_validate_algorithm_parameters_config(self):
```

Comment:
My initial unittests for DSWx_S1 passed with similar code.  As things got more complex I ended up having to copy the algorithm_parameters runconfig into the input_dir within my temporary test environment, then modify the path in the test config file in the test/data directory.  That solved some weird behavior that I was having.  Just a heads up for when you start validating the inputs.